examples/tv-app/linux/include/target-navigator/TargetNavigatorManager.cpp

```diff
@@ -58,31 +59,12 @@ std::vector<EmberAfNavigateTargetTargetInfo> TargetNavigatorManager::proxyGetTar
     return targets;
 }
 
-TargetNavigatorLaunchResponse TargetNavigatorManager::proxyNavigateTargetRequest(uint8_t target, std::string data)
+TargetNavigatorResponse targetNavigatorClusterNavigateTarget(uint8_t target, std::string data)
 {
     // TODO: Insert code here
-    TargetNavigatorLaunchResponse response;
-    response.data   = "data response";
+    TargetNavigatorResponse response;
+    char testData[] = "data response";
+    response.data   = (uint8_t *) testData;
     response.status = EMBER_ZCL_APPLICATION_LAUNCHER_STATUS_SUCCESS;
     return response;
```

Comment:
This is again returning a pointer-to-stack.  Can't do that.

Reply:
Updated.